### PR TITLE
Stopping at first error

### PR DESCRIPTION
**Pull request by wmathurin**
Right now: building the ios repo by running ant fails. However the exit value was not being checked by update.sh so it just kept going and as a result, test_force.js would think that everything went well when it didn't.

### CODE REVIEW

File: tools/update.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-#set -x
+#set -x  # turn trace on 
+set -e   # stop at first error
```
Comment (wmathurin) [inline tools/update.sh]: See: http://julipedia.meroh.net/2010/01/set-e-and-set-x.html

**Comment by bhariharan** [comment]
:+1:

**Comment by khawkins** [comment]
Works for me.  We can get more granular on error handling if `-e` becomes problematic (sometimes it does), but I think this is fine for the time being.

**Comment by wmathurin** [comment]
Regarding my last comment "Right now: building the ios repo by running ant fails." It's only true with 7.2 (or 7.3) but works fine with Xcode 7.1.

**Pull request merged**